lib/log.cpp

```diff
@@ -0,0 +1,65 @@
+#include <stdio.h>
+#include <stdarg.h>
+#include <syslog.h>
```

Comment:
I knew I wasn't familiar with this. Looks like it will also be an issue on Windows. 

No need to change the code though, as there is this: https://github.com/asankah/syslog-win32

Probably more shims like it out there. I'll keep a look out.